src/commands/ptzplayaudio.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const toNumber = require('./utils/toNumber.js');
+
+/**
+ * @type {Record<string, number>}
+ */
+const audioNameToIdMap = {
+  alarm: 37,
+  siren: 49,
+  emergency: 40,
+  trespassing: 43,
+  camera: 33,
+  hello: 1,
+  despacito: 0,
+  ringtone: 35,
+  dog: 44
+}
+
+/**
+ * @type {import('./types.d.ts').CommandRegister}
+ */
+module.exports = (controller) => {
+  return {
+    name: 'ptzplayaudio',
+    aliases: [
```

Comment:
Curious if there's an advantage to having `name` and `aliases` be separate vs just allowing `name` to be an array or a string. I can't really think of a difference other than semantics

Reply:
Only in that if we pull this data into the website, it'll be the "primary" name by which it is referenced -- we could also just name an array though as you say and use the first as that, might be cleaner.